index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Loan System</title>
+    <script src="https://kit.fontawesome.com/60f96566f6.js" crossorigin="anonymous"></script>
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav>
+        <div class="logo"><a href="index.html">My Logo</a></div>
+        <ul class="navbar">
+            <li class="nav-items"><a href="index.html">home</a></li>
+            <li class="nav-items"><a href="about.html">about</a></li>
+            <i class="fas fa-bars" onclick="displayMenu()"></i>   
+        </ul>
+        
+    </nav>
+    <div id='popup'></div>
+    <header>
+        <h5 class="first-text">"Hello! Dear Businessman"</h5>
+        <h1 class="second-text">We have the softare you need to take your business 
+            to next level</h1>
+        <p class="paragraph">We have develop the system you were looking for, wich will help you
+            track your loans, payments and reports with all the useful data you need.
+        </p>
+    </header>
```

Comment:
The section should look like the original design (you can use the color gradient for the text)
#### Original Design
![right1](https://user-images.githubusercontent.com/58376020/124035048-4787cc00-d9fc-11eb-93d4-535487eed824.png)

#### Your Project
![wrong1](https://user-images.githubusercontent.com/58376020/124035080-553d5180-d9fc-11eb-9904-d6260523a85f.png)